### ISSUE TITLE
Show rank medal evolution dates/graph rank medal change history

### ISSUE THREAD

**Issue by evillman** [odota/web#1806]
**Is your feature request related to a problem? Please describe.**
We should add a simple feature instead of MMR. As  MMR is long gone, we could add a new tab to players profiles, showing the player medal evolution for each season.

**Teachability, Documentation, Adoption, Migration Strategy**
It should be quite easy to achieve, since we already have the current medal for everyone that enabled public data exposure.

**Comment by howardchung** [comment]
We can add a database table to track rank history and check it for deltas on each rank check.  However I don't think there's a reliable way to determine season changes.

**Comment by evillman** [comment]
> We can add a database table to track rank history and check it for deltas on each rank check. However I don't think there's a reliable way to determine season changes.

Maybe we could set official Valve dates manually.

**Comment by GoldMath** [comment]
Actually the **MMR History** don't work at all since it actually impossible to track. 
It should be replaced by something else.

When visiting a profile you can actually see the MMR Medal Rank, example "Legend [4]". That data could be on a graph with a date and when a player rank up or down it's update.

_Was about to request the same, so "upvote"_

**Comment by howardchung** [comment]
Yeah, that is probably the best thing to do now with the available data. We'll need to create a new database table and store rank changes in that for each player.
